prow/cluster/build_kubeconfig.yaml

```diff
@@ -0,0 +1,101 @@
+# Kubeconfig provides access to each of our Kubernetes build clusters.
```

Comment:
This is a really nice change, I like the single KUBECONFIG
You might consider documenting how someone might generate/update these credentials. If a cluster has to get rebuilt or another one added.